src/main/java/org/apache/commons/imaging/formats/tiff/taginfos/TagInfoDirectory.java

```diff
@@ -16,14 +16,23 @@
  */
 package org.apache.commons.imaging.formats.tiff.taginfos;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 import org.apache.commons.imaging.formats.tiff.constants.TiffDirectoryType;
+import org.apache.commons.imaging.formats.tiff.fieldtypes.FieldType;
 
 /**
  * A LONG representing an offset to a TIFF directory.
  */
 public class TagInfoDirectory extends TagInfoLong {
+
+    static final List<FieldType> fieldList
```

Comment:
Let's leave this one `private` until we need to make it public? It appears to be used only on this class.